composer/muxers.csv

```diff
@@ -0,0 +1,11 @@
+id,muxer
```

Comment:
Naive question, will there be an additional csv for security protocol TLS/Noise, or is that being handled differently?

Reply:
will be another file, i.e. `sec.csv`:

```csv
id,protocol
rust-master,noise
rust-master, tls
rust-0.49.0,noise
rust-0.49.0,tls
```

that is then queried along the other ones.
The process of adding implementations can be improved later, either by creating a cli tool that does it or, getting another primary format, that is then parsed by the composer and converted to `sql`